aws/resource_aws_s3_bucket.go

```diff
@@ -377,6 +377,10 @@ func resourceAwsS3Bucket() *schema.Resource {
 										Set:      destinationHash,
 										Elem: &schema.Resource{
 											Schema: map[string]*schema.Schema{
+												"account": {
```

Comment:
I'd suggest renaming this to `account_id` to match similar places in the code base.
Also can add `ValidateFunc: validateAwsAccountId,`.

Reply:
Both these seem reasonable and will be updated on merge.